Model/lib/conifer/roles/conifer/templates/EbrcWebsiteCommon/appBase.html.j2

```diff
@@ -41,6 +41,8 @@
         {% endif %}
         {% if user_datasets_workspace is defined %}
         useUserDatasetsWorkspace: "true" === "{{ user_datasets_workspace.enabled|default('false')}}",
+        communityDatasetsEnabled: "{{ user_datasets_workspace.community|default('') }}",
+        showExtraMetadata: "{{ user_datasets_workspace.showExtraMetadata|default('') }}",
```

Comment:
Are these intentionally being included in this PR?

Reply:
Sort of.  Mix up with the branches.  But I think it's going to work out.